src/pkg/toolbox/image.go

```diff
@@ -0,0 +1,131 @@
+package toolbox
+
+import (
+	"errors"
+	"path/filepath"
+
+	"github.com/containers/toolbox/pkg/utils"
+	"github.com/sirupsen/logrus"
+)
+
+var (
+	// knownImages holds all known toolbox images
+	// These should work out of the box with all Toolbox commands
+	knownImages = map[string]Image{
+		"fedora-toolbox": {
+			"registry.fedoraproject.org",
+			"fedora-toolbox",
+			"latest",
```

Comment:
The `latest` part makes me a bit nervous. :)

I did `toolbox create` on my *F33* host, and got a *F34* container named `fedora-toolbox-latest`. Another point of confusion would be when the host gets upgraded, or the value of `latest` changes over time.